CenterForOpenScience/osf.io
# Adding post endpoints for registries moderation [ENG-2130]

## Purpose

Adding POST/PATCH endpoints for registries moderation
## Changes

Adding serializers. Modifying views.
## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify you can perform actions against the endpoint. Verify it doesn't allow you to make actions out of order
- Verify you can make requests. Verify you can update the comment

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

I don't think moderation docs are public

## Side Effects

N/A
## Ticket

https://openscience.atlassian.net/browse/ENG-2130
